Sources/OVP/Provider/OVPMediaProvider.swift

```diff
@@ -275,7 +275,6 @@ public enum OVPMediaProviderError: PKError {
                     }
                     
                     guard let baseEntry = ovpBaseEntryList?.objects?.last,
-                        let sources = ovpPlaybackContext?.sources,
```

Comment:
Why was this removed?
Can we continue the code if we have no source?
The MediaEntry is built on the source.